vmdb/db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Provisioning/Placement.class/__methods__/best_fit_amazon.rb

```diff
@@ -15,3 +15,43 @@
 #  best Availability Zone for you based on system health and available capacity. If you launch additional
 #  instances, only specify an Availability Zone if your new instances must be close to, or separated from,
 #  your running instances."
+#
+#
+# VPC & Subnet
+# For C4 and T2 instances we have to provide VPC and Subnet.
+# http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/t2-instances.html
+# Read section marked "EC2-VPC-only Support"
+# For other instance types EC2 provides the defaults.
+# This method checks the instance types and sets the VPC and subnet for T2 and C4 instances
+#
+
+VPC_INSTANCE_GROUPS = %w(C4 T2)
+
+def set_property(prov, image, list_method, property)
+ if prov.get_option(property.to_sym).nil?
```

Comment:
If you pass `property` as a symbol you do not need the `.to_sym` call here and all other string interpolation calls will work as well.